edenia/eden-spend-explorer
# feat(webapp): add loading to piechart when is empty

### Add loading to the piechart when is empty

### What does this PR do?

- Resolve #429 

### Steps to test

1. run the project and open localhost:3000 in your browser 
2. reload the website and when there isn't data the chart should show a loader
